livestyled/models/fulfilment_point.py

```diff
@@ -99,6 +100,7 @@ def __init__(
         self.position = position
         self.reference = reference
         self.external_id = external_id
+        self.audiences = audiences
```

Comment:
I think you need to do the same thing like `categories` below.

Reply:
You also need to update the `schema/fulfilment_point.py`, the Audience needs a schema like 
`categories = RelatedResourceField(schema=FulfilmentPointCategorySchema, many=True, missing=[])`. You need to attache an AudienceSchema to it.